common/build/eslint-config-fluid/package.json

```diff
@@ -26,6 +26,7 @@
     "eslint-plugin-editorconfig": "~3.2.0",
     "eslint-plugin-eslint-comments": "~3.2.0",
     "eslint-plugin-import": "~2.25.4",
+    "eslint-plugin-jsdoc": "~39.3.0",
```

Comment:
You won't be able to apply any fixes without also adding this to the individual packages, too. Might as well do it here to keep the code change PR really focused on just code changes.

Reply:
Is that preferable over just adding the dependency in the individual packages as they turn on rules? I can see arguments for both options, so I am happy to take whatever approach we prefer, just wanted to clarify :)

Reply:
From a maintenance standpoint it's easier if all the packages have the peer dependencies that they need. While we don't express it in package.json, the eslint config technically has a peer dependency on the plugins that it uses, so it's also a bit more correct.

For making the change, I usually go low-tech. I would just use VSCode to search for `"eslint-plugin-import": "~2.25.4",` in package.json and replace it with `"eslint-plugin-import": "~2.25.4","eslint-plugin-jsdoc": "~39.3.0",`, then run policy-check:fix to clean up all the package.jsons. Then update the lockfiles.

Reply:
Actually, I'm a bit confused here. The other packages should get the new dep automatically as a transitive dependency through the config package right? Why do they need to depend directly on the plugin?

Reply:
Hmm, I could have sworn the transitive dep was sufficient even for plugins. We had a similar pattern in the Whiteboard framework repo, but the plugin packages didn't leak out like this. Is this just a local development issue due to the way the symlinks work out?

Reply:
Happy to make the change here until we resolve this, but I think we can clean this up.

Reply:
There is a workaround that I attempted to integrate in #9562. When I went to apply the changes to downstream packages, though, I ran into problems when using fluid-build that caused a number of rules to throw exceptions, most notably unbound-method. I never had a chance to chase down the problem, but I think all the pieces are in place already to address it.

Reply:
I created #10410 which demonstrates another option for resolving this.